src/main/resources/static/css/styles.css

```
@@ -47,6 +165,75 @@ main {
+        justify-content: space-around;
+        list-style: none;
+        margin: 0;
+
+        @supports (corner-shape: squircle) {
+            corner-shape: squircle;
+            border-radius: 24px;
+        }
+    }
+
+    & ul > li {
+        margin: 0.25rem 1rem;
```

Stylelint (17.2.0)

[error] Unexpected duplicate "margin" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)